src/jdk.jcmd/share/classes/sun/tools/jmap/JMap.java

```diff
@@ -228,8 +228,8 @@ private static void dump(String pid, String options)
             } else if (subopt.startsWith("gz=")) {
                compress_level = subopt.substring("gz=".length());
                if (compress_level.length() == 0) {
-                    System.err.println("Fail: no number provided in option: '" + subopt + "'");
-                    usage(1);
+                   System.err.println("Fail: no number provided in option: '" + subopt + "'");
+                   usage(1);
```

Comment:
Actually lines 231 and 232 have the correct indentation. It's 229, 230, and 233 that are wrong. They need to be indented one more space.